src/components/grid-aware/Footer/InfoBlock/InfoBlock.module.css

```
@@ -0,0 +1,44 @@
+/* Background bleed */
+
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+.infoBlock {
+  background-color: var(--color-midnight-blue);
+  color: var(--color-gray-300);
+  display: grid;
+  font-family: var(--font-family-lato);
+  font-size: 14px;
+  font-weight: 300;
```

According to the designs, I think this is supposed to be 400. Note that we only load 400 and 700 for Lato anyway, so you won't actually see a difference when you change this, but it's a good idea to request exactly what the design requests.

```suggestion
  font-weight: 400;
```

---

ok